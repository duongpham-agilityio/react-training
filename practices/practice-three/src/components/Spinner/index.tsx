import { Spinner, Square } from '@chakra-ui/react';
import { memo } from 'react';

const Loading = () => (
  <Square size="full">
    <Spinner />
  </Square>
);

export default memo(Loading);
