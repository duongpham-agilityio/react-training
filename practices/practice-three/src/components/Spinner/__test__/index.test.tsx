import { render } from '@testing-library/react';

// Components
import Spinner from '..';

describe('Spinner', () => {
  const { container } = render(<Spinner />);

  expect(container).toMatchSnapshot();
});
