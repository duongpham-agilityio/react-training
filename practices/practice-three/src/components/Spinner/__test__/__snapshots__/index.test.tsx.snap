// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[` 1`] = `
<div>
  <div
    class="css-1ics0p0"
  >
    <div
      class="chakra-spinner css-1y7joxr"
    >
      <span
        class="css-8b45rq"
      >
        Loading...
      </span>
    </div>
  </div>
</div>
`;
